src/common/chat-participants/powerpages/PowerPagesChatParticipant.ts

```diff
@@ -113,25 +112,42 @@ export class PowerPagesChatParticipant {
             };
         }
 
-        const userPrompt = request.prompt;
+        if (request.command) {
+            //TODO: Handle command scenarios
 
-        if (!userPrompt) {
+        } else {
 
-            //TODO: Show start message
+            const userPrompt = request.prompt;
 
-            return {
-                metadata: {
-                    command: ''
-                }
-            };
-        }
+            if (!userPrompt) {
 
-        //TODO: Handle form and list scenarios
-        const llmResponse = await sendApiRequest([{ displayText: userPrompt, code: '' }], { dataverseEntity: '', entityField: '', fieldType: '' }, this.orgID, intelligenceApiToken, '', '', [], this.telemetry, intelligenceEndpoint, geoName);
+                //TODO: String approval is required
+                stream.markdown(NO_PROMPT_MESSAGE);
```

Comment:
what is this scenario here?

Reply:
It was for dataverse integration for form and list